library/core/registry/src/main/java/org/quiltmc/qsl/registry/impl/dynamic/DynamicMetaRegistryImpl.java

```diff
@@ -55,4 +63,8 @@ public static <E> void registerSynced(RegistryKey<? extends Registry<E>> ref, Co
 	public static void freeze() {
 		frozen = true;
 	}
+
+	static boolean isFrozen() {
+		return frozen;
```

Comment:
just a comment: hnnghhngh i hate the lack of a `this.` there but i can't fight fateeeeeeeee, it's a static field ;-;